### DISCUSSION TITLE
The game crashed whilst initializing game Error: java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'litematica'!

### DISCUSSION THREAD

**Discussion by ghost** [FabricMC/fabric-api#2762] [Player support] [answered]
I have tried opening Minecraft with Fabric loader to play with Litematica mod, but I'm met with this, ''The game crashed whilst initializing game Error: java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'litematica'!'' I also use other mods such as effortless build, fabric api, Falling tree, litematica, malilib, meteor client. Of course, all of these mods are fabric and 1.19.3.

---- Minecraft Crash Report ----
// Don't do that.

Time: 2022-12-15 22:25:51
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'litematica'!
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at net.minecraft.class_310.<init>(class_310.java:465)
	at net.minecraft.client.main.Main.method_44604(Main.java:205)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: net.fabricmc.loader.api.EntrypointException: Exception while loading entries for entrypoint 'main' provided by 'litematica'
	at net.fabricmc.loader.impl.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:56)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47)
	... 8 more
Caused by: net.fabricmc.loader.api.LanguageAdapterException: Class fi.dy.masa.litematica.Litematica cannot be cast to net.fabricmc.api.ModInitializer!
	at net.fabricmc.loader.impl.util.DefaultLanguageAdapter.create(DefaultLanguageAdapter.java:63)
	at net.fabricmc.loader.impl.entrypoint.EntrypointStorage$NewEntry.getOrCreate(EntrypointStorage.java:117)
	at net.fabricmc.loader.impl.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:53)
	... 9 more

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at net.minecraft.class_310.<init>(class_310.java:465)

-- Initialization --
Details:
	Modules: 
		ADVAPI32.dll:Advanced Windows 32 Base API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		COMCTL32.dll:User Experience Controls Library:6.10 (WinBuild.160101.0800):Microsoft Corporation
		CRYPT32.dll:Crypto API32:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTBASE.dll:Base cryptographic API DLL:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTSP.dll:Cryptographic Service Provider API:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		DBGHELP.DLL:Windows Image Helper:10.0.19041.867 (WinBuild.160101.0800):Microsoft Corporation
		DNSAPI.dll:DNS Client API DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		GDI32.dll:GDI Client DLL:10.0.19041.2130 (WinBuild.160101.0800):Microsoft Corporation
		IMM32.DLL:Multi-User Windows IMM32 API Client DLL:10.0.19041.2193 (WinBuild.160101.0800):Microsoft Corporation
		IPHLPAPI.DLL:IP Helper API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		KERNEL32.DLL:Windows NT BASE API Client DLL:10.0.19041.1741 (WinBuild.160101.0800):Microsoft Corporation
		KERNELBASE.dll:Windows NT BASE API Client DLL:10.0.19041.1741 (WinBuild.160101.0800):Microsoft Corporation
		MpOav.dll:IOfficeAntiVirus Module:4.18.2211.5 (WinBuild.160101.0800):Microsoft Corporation
		NLAapi.dll:Network Location Awareness 2:10.0.19041.2193 (WinBuild.160101.0800):Microsoft Corporation
		NSI.dll:NSI User-mode interface DLL:10.0.19041.610 (WinBuild.160101.0800):Microsoft Corporation
		NTASN1.dll:Microsoft ASN.1 API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		Ole32.dll:Microsoft OLE for Windows:10.0.19041.1320 (WinBuild.160101.0800):Microsoft Corporation
		OleAut32.dll:OLEAUT32.DLL:10.0.19041.985 (WinBuild.160101.0800):Microsoft Corporation
		PSAPI.DLL:Process Status Helper:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		Pdh.dll:Windows Performance Data Helper DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		RPCRT4.dll:Remote Procedure Call Runtime:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		SHCORE.dll:SHCORE:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		SHELL32.dll:Windows Shell Common Dll:10.0.19041.964 (WinBuild.160101.0800):Microsoft Corporation
		USER32.dll:Multi-User Windows USER API Client DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		USERENV.dll:Userenv:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		VCRUNTIME140.dll:Microsoft® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		VERSION.dll:Version Checking and File Installation Libraries:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		WINHTTP.dll:Windows HTTP Services:10.0.19041.2075 (WinBuild.160101.0800):Microsoft Corporation
		WINMM.dll:MCI API DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		WS2_32.dll:Windows Socket 2.0 32-Bit DLL:10.0.19041.1081 (WinBuild.160101.0800):Microsoft Corporation
		WSOCK32.dll:Windows Socket 32-Bit DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		Wldp.dll:Windows Lockdown Policy:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		amsi.dll:Anti-Malware Scan Interface:10.0.19041.2075 (WinBuild.160101.0800):Microsoft Corporation
		bcrypt.dll:Windows Cryptographic Primitives Library:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		bcryptPrimitives.dll:Windows Cryptographic Primitives Library:10.0.19041.1415 (WinBuild.160101.0800):Microsoft Corporation
		clbcatq.dll:COM+ Configuration Catalog:2001.12.10941.16384 (WinBuild.160101.0800):Microsoft Corporation
		combase.dll:Microsoft COM for Windows:10.0.19041.1320 (WinBuild.160101.0800):Microsoft Corporation
		dbgcore.DLL:Windows Core Debugging Helpers:10.0.19041.2311 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc.DLL:DHCP Client Service:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc6.DLL:DHCPv6 Client:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		fwpuclnt.dll:FWP/IPsec User-Mode API:10.0.19041.1503 (WinBuild.160101.0800):Microsoft Corporation
		gdi32full.dll:GDI Client DLL:10.0.19041.2364 (WinBuild.160101.0800):Microsoft Corporation
		java.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		javaw.exe:OpenJDK Platform binary:17.0.3.0:Microsoft
		jemalloc.dll
		jimage.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		jli.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		jna13737162215153797913.dll:JNA native library:6.1.4:Java(TM) Native Access (JNA)
		jsvml.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		jvm.dll:OpenJDK 64-Bit server VM:17.0.3.0:Microsoft
		kernel.appcore.dll:AppModel API Host:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		lwjgl.dll
		management.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		management_ext.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		msvcp140.dll:Microsoft® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		msvcp_win.dll:Microsoft® C Runtime Library:10.0.19041.789 (WinBuild.160101.0800):Microsoft Corporation
		msvcrt.dll:Windows NT CRT DLL:7.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		mswsock.dll:Microsoft Windows Sockets 2.0 Service Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		napinsp.dll:E-mail Naming Shim Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		ncrypt.dll:Windows NCrypt Router:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		net.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		nio.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		ntdll.dll:NT Layer DLL:10.0.19041.1741 (WinBuild.160101.0800):Microsoft Corporation
		perfos.dll:Windows System Performance Objects DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		pnrpnsp.dll:PNRP Name Space Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		profapi.dll:User Profile Basic API:10.0.19041.844 (WinBuild.160101.0800):Microsoft Corporation
		rasadhlp.dll:Remote Access AutoDial Helper:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		rsaenh.dll:Microsoft Enhanced Cryptographic Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		sechost.dll:Host for SCM/SDDL/LSA Lookup APIs:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		shlwapi.dll:Shell Light-weight Utility Library:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		sunmscapi.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		ucrtbase.dll:Microsoft® C Runtime Library:10.0.19041.789 (WinBuild.160101.0800):Microsoft Corporation
		vcruntime140_1.dll:Microsoft® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		verify.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		win32u.dll:Win32u:10.0.19041.2364 (WinBuild.160101.0800):Microsoft Corporation
		windows.storage.dll:Microsoft WinRT Storage API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		winrnr.dll:LDAP RnR Provider DLL:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		wshbth.dll:Windows Sockets Helper DLL:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		zip.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
Stacktrace:
	at net.minecraft.client.main.Main.method_44604(Main.java:205)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- System Details --
Details:
	Minecraft Version: 1.19.3
	Minecraft Version ID: 1.19.3
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 1564375920 bytes (1491 MiB) / 2080374784 bytes (1984 MiB) up to 2147483648 bytes (2048 MiB)
	CPUs: 16
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 9 5900HS with Radeon Graphics        
	Identifier: AuthenticAMD Family 25 Model 80 Stepping 0
	Microarchitecture: Zen 3
	Frequency (GHz): 3.29
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 16
	Graphics card #0 name: AMD Radeon(TM) Graphics
	Graphics card #0 vendor: Advanced Micro Devices, Inc. (0x1002)
	Graphics card #0 VRAM (MB): 512.00
	Graphics card #0 deviceId: 0x1638
	Graphics card #0 versionInfo: DriverVersion=30.0.13002.1001
	Graphics card #1 name: NVIDIA GeForce RTX 3060 Laptop GPU
	Graphics card #1 vendor: NVIDIA (0x10de)
	Graphics card #1 VRAM (MB): 4095.00
	Graphics card #1 deviceId: 0x2520
	Graphics card #1 versionInfo: DriverVersion=30.0.15.1278
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 3.20
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 3.20
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 18208.35
	Virtual memory used (MB): 11649.04
	Swap memory total (MB): 2432.00
	Swap memory used (MB): 47.00
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		effortless: Effortless Fabric 1.5.2
		fabric-api: Fabric API 0.69.0+1.19.3
			fabric-api-base: Fabric API Base 0.4.19+8d1895cf85
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.18+49abcf7e85
			fabric-biome-api-v1: Fabric Biome API (v1) 12.0.0+689f5e7185
			fabric-block-api-v1: Fabric Block API (v1) 1.0.3+12bfe4ea85
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.28+c6af733c85
			fabric-client-tags-api-v1: Fabric Client Tags 1.0.9+49abcf7e85
			fabric-command-api-v1: Fabric Command API (v1) 1.2.19+f71b366f85
			fabric-command-api-v2: Fabric Command API (v2) 2.1.15+49abcf7e85
			fabric-commands-v0: Fabric Commands (v0) 0.2.36+df3654b385
			fabric-containers-v0: Fabric Containers (v0) 0.1.44+df3654b385
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.4.12+49abcf7e85
			fabric-convention-tags-v1: Fabric Convention Tags 1.2.0+8e4e694f85
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.12+aeb40ebe85
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 11.1.0+112a38be85
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.39+48349a3f85
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.5.5+b83334a085
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.37+422b77fb85
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.39+df3654b385
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.28+2894d6df85
			fabric-item-api-v1: Fabric Item API (v1) 2.1.5+4d2c606385
			fabric-item-group-api-v1: Fabric Item Group API (v1) 2.1.7+bdbead0185
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.28+aaaf9d3385
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.26+df3654b385
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.8+2894d6df85
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.18+75e9821185
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.22+9e7660c685
			fabric-message-api-v1: Fabric Message API (v1) 5.0.12+6ede1da985
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.29+49abcf7e85
			fabric-models-v0: Fabric Models (v0) 0.3.25+11ba9c3b85
			fabric-networking-api-v1: Fabric Networking API (v1) 1.2.14+d9ac3c3985
			fabric-networking-v0: Fabric Networking (v0) 0.3.31+df3654b385
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 5.2.0+e9dee00485
			fabric-particles-v1: Fabric Particles (v1) 1.0.18+fbde993d85
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 2.0.1+2608564685
			fabric-renderer-api-v1: Fabric Renderer API (v1) 2.1.0+2063beff85
			fabric-renderer-indigo: Fabric Renderer - Indigo 0.6.23+11ba9c3b85
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.28+df3654b385
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.23+48349a3f85
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.16+3067f46385
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.31+df3654b385
			fabric-rendering-v1: Fabric Rendering (v1) 1.11.7+49abcf7e85
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.2.0+280be3ab85
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.10.3+a10d22bd85
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.39+1d4d0b5785
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.10+689f5e7185
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.6+75e9821185
			fabric-transfer-api-v1: Fabric Transfer API (v1) 2.1.12+82a2701785
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 2.1.0+0efcd39b85
		fabricloader: Fabric Loader 0.14.11
		fallingtree: FallingTree 3.11.1
		java: OpenJDK 64-Bit Server VM 17
		litematica: Litematica 0.13.0
		malilib: MaLiLib 0.14.0
		meteor-client: Meteor Client 0.5.2-1715
			baritone-meteor: Baritone 1.19.3-SNAPSHOT
		minecraft: Minecraft 1.19.3
	Launched Version: fabric-loader-0.14.11-1.19.3
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: Unknown
	Window size: <not initialized>
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: <disabled>
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	CPU: <unknown>

-- Meteor Client --

Version: 0.5.2
Dev Build: 1715

**Comment by warjort** [comment, marked as answer]
````
Caused by: net.fabricmc.loader.api.LanguageAdapterException: Class fi.dy.masa.litematica.Litematica cannot be cast to net.fabricmc.api.ModInitializer!
````
Looks like some issue with the Litematica mod.
Check you have the latest version then contact the mod author.

**Comment by maruohon** [comment]
This is/was a bug in Baritone. Either update it or remove it.

**Comment by ghost** [comment]
Deleting the old mod configs from "config" folder in .minecraft solved the issue for me
